### ISSUE TITLE
Download Eslint and TypeScript in `dev.js` script

### ISSUE THREAD

**Issue by jespertheend** [rendajs/Renda#139]
At the moment TypeScript is already being downloaded, but once #127 lands this will no longer be the case.
However, during development it can be nice to have a downloaded version of both TypeScript and eslint as it will allow IDE extensions to use specific versions of these.
We should add an extra flag to the `dev.js` script to skip these downloads in order to speed up ci.
